docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md

```diff
@@ -0,0 +1,128 @@
+# Update-AriaNetworksvCenterDataSourceCredentials
+
+## Synopsis
+
+Update a configured vCenter Server data source's credentials in VMware Aria Operations for Networks.
+
+## Syntax
+
+```powershell
+Update-AriaNetworksvCenterDataSourceCredentials [-id] <String> [-collectorId] <String> [[-fqdn] <String>]
+ [-nickname] <String> [-username] <String> [-password] <String> [<CommonParameters>]
+```
+
+## Description
+
+The `Update-AriaNetworksvCenterDataSourceCredentials` cmdlet allows a user to update a previously configured vCenter Servers credentials in VMware Aria Operations for Networks.
+
+## Examples
+
+### Example 1
+
+```powershell
+Update-AriaNetworksvCenterDataSourceCredentials -id 15832:902:2623605245375371420 -CollectorId 15832:901:1711011916294613031 -fqdn sfo-m01-vc01.sfo.rainpole.io -username svc-inv-vsphere -password VMw@re1!  -nickname "sfo-m01-vc01 - Management Domain vCenter Server"
+```
+
+This example updates a configured vCenter Server with updated values as supplied in VMware Aria Operations for Networks.

```

Comment:
```suggestion
This example updates a configured vCenter Server with updated credentials as supplied in VMware Aria Operations for Networks.

```

Reply:
```suggestion
This example updates the credentials for vCenter Server data source in VMware Aria Operations for Networks.

```

Reply:
Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/7344af3d25523db031599dbdd47c90a7d78ba0d9